packages/asc-ui/src/components/TopTaskButton/TopTaskButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import TopTaskButtonStyle, { Props } from './TopTaskButtonStyle'
+import Icon from '../Icon'
+
+const TopTaskButton: React.FC<Props> = ({
```

Comment:
Then consider to use a Button or Link as a default, we can type this like:
`React.FC<Props & React.AnchorHTMLAttributes<HTMLAnchorElement> &  React.ButtonHTMLAttributes<HTMLButtonElement>>`

Reply:
we cannot make it an anchor because otherwise you cannot use Router Links, but we can use a Button instead

Reply:
it is a button now